chapters/traversal.md

```diff
@@ -1,83 +1,85 @@
 # Understand Jina Recursive Document Representation
 
-In Jina, each Document is represented as a recursive representation (tree).
+## General overview
 
-A rooted recursive representation has a root node and every node has X children.
-In Jina, the root node is the document itself, while the *left* & *right* children are referred to as *chunks* and *matches* respectively.
+If you add a text document or an image to the Jina database, Jina indexes it as a new document. Internally, each Document is represented as a rooted recursive representation (tree).
```

Comment:
I would rather not refer to Jina as a database and the rooted representation is optional, so please replace line 5 with. 

When you index a Document in Jina, it can be represented as a rooted recursive representation (tree).

Reply:
are we sure we want to refer to Jina as database? I think is missleading

Reply:
No we shouldn't, I told Frank to remove that.

Reply:
Alright, I remove the line as suggested